File: examples/quran/chapters/node/listChapters.js
const { QuranService } = require("micro-js-client/quran");

// List the Chapters (surahs) of the Quran
async function listChapters() {
  let quranService = new QuranService(process.env.MICRO_API_TOKEN);
  let rsp = await quranService.chapters({
    language: "en",
  });
  console.log(rsp);
}

listChapters();

File: examples/quran/chapters/go/listChapters.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/quran"
)

// List the Chapters (surahs) of the Quran
func ListChapters() {
	quranService := quran.NewQuranService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := quranService.Chapters(&quran.ChaptersRequest{
		Language: "en",
	})
	fmt.Println(rsp, err)
}
